javascript/ql/src/Summary/TaintSources.ql

```diff
@@ -0,0 +1,16 @@
+/**
+ * @name Taint sources
+ * @description Sources of untrusted input.
+ * @kind problem
+ * @problem.severity info
+ * @id js/summary/taint-sources
+ * @tags summary
+ * @precision medium
+ */
+
+import javascript
+import meta.internal.TaintMetrics
+
+from RemoteFlowSource node
+where node = relevantTaintSource()
```

Comment:
`relevantTaintSource` has return-type `DataFlow::Node`, so I think the `node` here should have that type. 

```suggestion
from DataFlow::Node node
where node = relevantTaintSource()
```


Or `relevantTaintSource` could be changed to have the return-type `RemoteFlowSource`.   
`DOM::locationSource()` is a `RemoteFlowSource` anyway, so `relevantTaintSource` could be simplified in the same go.